lib/google-apis/auth-client.js

```diff
@@ -83,18 +137,25 @@ AuthClient.prototype._getApplicationDefaultCredentials = function() {
 
     if (err) {
 
```

Comment:
style nit: avoid unnecessary blank link after block start, here and elsewhere in the file. This can be fixed in a follow-on (don't pollute this particular PR). Aside: could clang-format be configured to catch this?